test/apps/iodemo/io_demo.cc

```diff
@@ -667,6 +682,25 @@ class DemoServer : public P2pDemoCommon {
         send_io_message(conn, IO_READ_COMP, msg->sn, 0, opts().validate);
     }
 
+    void handle_io_am_read_request(UcxConnection* conn, const iomsg_t *msg) {
+        VERBOSE_LOG << "sending AM IO read data";
+        assert(opts().max_data_size >= msg->data_size);
+
+        IoMessage *m = _io_msg_pool.get();
+        m->init(IO_READ_COMP, msg->sn, msg->data_size, opts().validate);
+
+        BufferIov *iov = _data_buffers_pool.get();
+        iov->init(msg->data_size, _data_chunks_pool, msg->sn, opts().validate);
+
+        SendCompleteCallback *cb = _send_callback_pool.get();
+        cb->init(iov, &_curr_state.read_count, m);
+
+        assert(iov->size() == 1);
+
+        conn->send_am(m->buffer(), opts().iomsg_size, (*iov)[0].buffer(),
```

Comment:
pls add comment that we send IOMSG as header and the first IOV as payload
(multi iov not supported now)?

Reply:
ok . multi iov is not supported (there is an assert above this line)

Reply:
added comment